app/modules/ads/index.js

```diff
@@ -46,16 +44,10 @@ class Ads {
       return adsPromise;
     }
 
-    if (typeof FastBoot !== 'undefined') {
```

Comment:
It's causing a memory leak on the server side, isn't it? Am I missing something?

Reply:
There should be no memory leaks now.